web/src/app/app.component.ts

```diff
@@ -70,7 +71,7 @@ export class AppComponent implements OnInit {
         private _iconRegistry: MatIconRegistry,
         private _domSanitizer: DomSanitizer,
         private authService: AuthenticationService,
-        private spinnerService: SpinnerService
+        private activityService: ActivityService
```

Comment:
We are following "," at the end ?

Reply:
Yes we are 👍 . I will fix this now and I will also create an issue to have a lint rule for this

Reply:
https://github.com/DashboardHub/PipelineDashboard/issues/1243

Reply:
okay please let me know once all reviews are fixed